app/javascript/leftSidebar/TagsFollowed.jsx

```diff
@@ -12,20 +12,15 @@ export const TagsFollowed = ({ tags = [] }) => {
     <Fragment>
       {tags.map(({ name, id, points }) =>
         points >= 1 ? (
-          <div
+          <a
```

Comment:
Don't we want to use the `Link` component instead of adding the classes manually to the `a`?

Reply:
hah, yes - im an idiot :D i was just quickly going through everything and kinda forget we've just built this component :D